src/containers/target-pane.jsx

```diff
@@ -189,9 +189,15 @@ class TargetPane extends React.Component {
                 }, targetId);
             } else if (dragInfo.dragType === DragConstants.BACKPACK_SOUND) {
                 this.props.vm.addSound({
-                    md5: dragInfo.payload.body,
+                    md5: dragInfo.payload.bodiy,
```

Comment:
This looks like a typo... or maybe just a stray keypress since I don't think you were trying to do anything here?

Reply:
This still looks like a typo... maybe the GitHub comment-minifier has worked against us already.